src/char/hammerdin.py

```diff
@@ -114,7 +114,7 @@ def kill_council(self) -> bool:
         # Check out the node screenshot in assets/templates/trav/nodes to see where each node is at
         atk_len = self._char_config["atk_len_trav"]
         # Go inside and hammer a bit
-        self._pather.traverse_nodes([228, 229], self, time_out=2.5, force_tp=True)
+        self._pather.traverse_nodes([228, 229], self, time_out=2.5, force_tp=True, do_pre_move=False)
```

Comment:
Did you test with a teleporting pala that we are on teleport when this is called. It should, but I'd rather know it was tested.

Reply:
> 

Nope, only tested it walking. I will have to make a hero edited character to test

Reply:
> Did you test with a teleporting pala that we are on teleport when this is called. It should, but I'd rather know it was tested.

I am creating a new pull request that addresses this without affecting Enigma Hammerdins. Closing this one because my newer change is in a diff fork.